### DISCUSSION TITLE
can pynecone be used for Mobile app in future? like cross-platform

### DISCUSSION THREAD

**Discussion by ahmad-shahi** [reflex-dev/reflex#1380] [Ideas]
It is not a bug, i just wonder, can pynecone be used for mobile app in future development? Thanks for good work

**Comment by DanielFrantes** [comment]
Please move to discussion.
But I think it cannot be used in mobile phone.

**Comment by masenf** [comment]
We have plans to support as many use cases as possible, but currently don't have any time frame for when mobile will be supported.

**Reply by abcnow** [reply]
any update on this @masenf ? I think it will be great if we can pack this for desktop and mobile.. it would open lots of possibilities.. though I love flutter.. it can be painful to deal with ai for example...Only python is the way..

**Reply by picklelo** [reply]
We have no immediate plans for this as we're focusing on making Reflex work well for web apps first (and we still have a long list of issues to work through for that).

We would welcome outside contributors kickstarting this and would support in anyway we can, but we don't have bandwidth for this at the moment.

**Comment by camstuart** [comment]
I think capacitor is the path to least resistance, see: https://github.com/orgs/reflex-dev/discussions/3152
